Modules/_templatelibmodule.c

```diff
@@ -0,0 +1,38 @@
+/* interpreter-internal types for string.templatelib */
```

Comment:
See my comment above.
I don't see any advantage to added this module over adding the two classes to the `types` module.

Reply:
This was the SC's decision.